File: 2026/05-May/27-Wednesday/solution.java
import java.util.*;

class Solution {
    public int numberOfSpecialChars(String word) {
        Set<Character> lower = new HashSet<>();
        Set<Character> upper = new HashSet<>();

        for(char ch : word.toCharArray()) {
            if(Character.isLowerCase(ch))
                lower.add(ch);
            else
                upper.add(Character.toLowerCase(ch));
        }

        lower.retainAll(upper);
        return lower.size();
    }
}

File: 2026/05-May/27-Wednesday/solution.js
var numberOfSpecialChars = function(word) {
    let lower = new Set();
    let upper = new Set();

    for(let ch of word){
        if(ch >= 'a' && ch <= 'z')
            lower.add(ch);
        else
            upper.add(ch.toLowerCase());
    }

    let ans = 0;

    for(let ch of lower){
        if(upper.has(ch))
            ans++;
    }

    return ans;
};
